src/main/scala/no/ndla/learningpathapi/service/UpdateServiceComponent.scala

```diff
@@ -133,13 +133,27 @@ trait UpdateServiceComponent {
             searchIndexService.indexDocument(updatedLearningPath)
           } else if (existing.isPublished) {
             searchIndexService.deleteDocument(updatedLearningPath)
+            deleteIsBasedOnReference(learningPathId, updatedLearningPath)
           }
 
           Some(converterService.asApiLearningpath(updatedLearningPath, Option(owner)))
         }
       }
     }
 
+    private def deleteIsBasedOnReference(learningPathId: Long, updatedLearningPath: domain.LearningPath) = {
+      if (updatedLearningPath.isPrivate || updatedLearningPath.isDeleted) {
```

Comment:
Tja. Jeg sjekker utenfor om den eksisterende stien om den var publisert. Hvis vi i fremtiden skulle innføre en ny status (som den vi hadde tidligere, synlig, men ikke søkbar) må vi vel sjekke om stien faktisk blir gjort utilgjengelig for allmenheten (slettet/privat) selvom den var publisert? Er forsåvidt enig at dagens løsning så vil den oppdaterte stien være privat/slettet hvis den var publisert og man gjør en oppdatering på status. Så hovedgrunnen til at jeg har denne sjekken er fordi jeg være sikker på at den nye stien er privat/slettet før jeg eventuelt fjerner `isBasedOn`. Gir dette mening eller er jeg helt ute på jordet?